src/components/typography/Heading.tsx

```diff
@@ -0,0 +1,52 @@
+import { Text, StyleSheet } from "react-native";
+
+import { global } from "@/constants/DesignTokens";
+
+type HeadingProps = {
+  size?:
```

Comment:
You can put that code on a single line if you want to

```suggestion
  size?: `heading${1 | 2 | 3 | 4 | 5 | 6}`
```

Reply:
I changed it to be like this.